rules/test_host_app/BUILD.bazel

```diff
@@ -23,5 +23,7 @@ exports_files([
         "10.0",
         "11.0",
         "12.0",
+        "13.0",
```

Comment:
Is this something that we should be able to define externally so we don't have to make library bumps each time a new xcode comes out?

Reply:
There's no way to do that